dts/bindings/adc/telink,b91-adc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2022, Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+description: Telink B91 ADC node
+
+compatible: "telink,b91-adc"
+
+include: adc-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    "#io-channel-cells":
+      const: 1
+
+    sample-freq:
+      type: int
+      required: true
+      enum:
+        - 23000
+        - 48000
+        - 96000
+
+    vref-internal-mv:
+      type: int
+      required: true
+      enum:
+        - 900
+        - 1200
```

Comment:
nit: please add description for these props

Reply:
Done.